test_olog.py

```diff
@@ -123,6 +156,13 @@ def test_put_tag():
     cli.put_tag(TAG)
 
 
+@vcr.use_cassette()
+def test_put_tag_with_error():
+    # vcr will return a wrong tag
```

Comment:
It took me awhile to figure out why this was expected to fail, since the body of the test code is of course identical to the test above it. This is clever, but it might help to explain more.
```suggestion
    # vcr will return a wrong tag because the recorded
    # response has been manually edited to be inconsistent
    # with the request to exercise this code path
```

Reply:
Right. thanks for this suggestion.